frappe/core/doctype/log_settings/log_settings.py

```diff
@@ -2,63 +2,188 @@
 # Copyright (c) 2020, Frappe Technologies and contributors
 # License: MIT. See LICENSE
 
+from typing import Protocol, runtime_checkable
+
 import frappe
 from frappe import _
+from frappe.model.base_document import get_controller
 from frappe.model.document import Document
-from frappe.query_builder import DocType, Interval
-from frappe.query_builder.functions import Now
+from frappe.utils import cint
+from frappe.utils.caching import site_cache
+
+DEFAULT_LOGTYPES_RETENTION = {
+	"Error Log": 30,
+	"Activity Log": 90,
+	"Email Queue": 30,
+	"Route History": 90,
```

Comment:
I am unsure about this, should we add this by default? 👀 

I think it should be fine cause if activity log is okay to delete (which contains auth related logs) then route history should be fine too. 


@gavindsouza @surajshetty3416

Reply:
Although it's unreliable, maybe let's not enable Route History deletion by default...Primarily because it'd be a breaking change 😅. Another way to go about this, is set a higher retention limit - say 1 year?

Reply:
Removing route history from defaults, users can add it back on their own if they want it.

Reply:
Error snapshot and scheduled job logs are fair game IMO, they are similar to error logs and aren't required for any sort of "forensics" after 90ish days.